spec/requests/post_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+  subject do
+    Post.new(author_id: first_user.id, text: 'This is a test post.', likes_counter: 0, comments_counter: 0)
+  end
+  before { subject.save }
+
+  it 'is not valid without a text' do
+    subject.text = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without an author' do
+    subject.author_id = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a likes counter' do
+    subject.likes_counter = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a comments counter' do
+    subject.comments_counter = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Please test the show and index actions using the right matchers as requested in the project.
>describe 'GET /index' do
  ...
end

>describe 'GET /show' do
  ...
end
![image](https://user-images.githubusercontent.com/91486791/223791267-3007530d-43c8-48bf-b41d-95f07fbf9ee4.png)